src/components/header.tsx

```diff
@@ -34,7 +34,10 @@ export default function Header() {
 							onClick={ () => getIpcApi().openSiteURL( site.id ) }
 							variant="link"
 						>
-							{ __( 'Open site' ) }
+							{
+								// translators: "Open site" refers to the action of opening a website
```

Comment:
Would it be clearer?

```suggestion
								// translators: "Open site" refers to the action, like "to open site"
```

Reply:
haha that's what I used first but Chat GPT said to make it shorter 😅 I agree that it is better this way though